### ISSUE TITLE
Feature request - copy and paste calendar events

### ISSUE THREAD

**Issue by ebeyer** [nextcloud/server#14081]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by MorrisJobke** [comment]
> **Is your feature request related to a problem? Please describe.**
> A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
> 
> **Describe the solution you'd like**
> A clear and concise description of what you want to happen.
> 
> **Describe alternatives you've considered**
> A clear and concise description of any alternative solutions or features you've considered.
> 
> **Additional context**
> Add any other context or screenshots about the feature request here.

This is the default text. Do you mind to add a bit more context? Then it would also be better suited to be located in the calendar repo: https://github.com/nextcloud/calendar

**Comment by ebeyer** [comment]
On Thu, 2019-02-07 at 13:09 +0000, Morris Jobke wrote:
> > Is your feature request related to a problem? Please describe.
> > A clear and concise description of what the problem is. Ex. I'm
> > always frustrated when [...]

I often need to manually input my work schedule for the next couple of months.  The schedule is published on paper.  The days I work vary from week to week.  In the past, I have used the copy/paste feature in iCal to input these commitments (sometimes hundreds at a time), but this is not currently possible in nextcloud's web interface.

> > Describe the solution you'd like
> > A clear and concise description of what you want to happen.

I would like to be able to select an appointment, copy it and then be able to paste it several times throughout the rest of the month.

> > Describe alternatives you've considered

I could continue to use iCal, but I would like to finalize my divorce from mac OS.  I have used thunderbird lightning but it does not always connect nicely to my calDAV.  I also use Evolution, but this is less elegant than simply using nextcloud directly.

Is this better?
Thank you for your time and consideration.
EB
